ISSUE_TEMPLATE/5-spike-research.md

```diff
@@ -0,0 +1,49 @@
+---
+name: "🔬 Spike/Research Request"
+about: "Suggest a theme of research for this project"
+title: ""
+labels: "Type: Research"
+---
+
+
+## Spike/Research Request
+
+## Context
+<!-- Outline the reason and context for the spike.
+
+Example: Team needs to use a database to store data for application developed
+during this subject and must learn to use the Firebird DBMS tools to be able to
+script the creation and population of the database. -->
+*
+
+## Knowledge Gap
+
+## Skill Gap
+
+## Technology Gap
+<!-- At least one of the above "Gap"s should be present -->
+*
+
+<!-- Provide details of the appropriate gaps related to this spike.
+
+Example: The team is not familiar with the new cloud structure. Need to install and be familiar with the following structure before going on:
+- ACME containers
+- FlameRobin GUI admin tool -->
+
+## Goals/Deliverables
+<!-- What are the goals and deliverables of this spike?
+Examples:
+- Clearing points to decide which technology to use among available options;
```

Comment:
We could use `- [ ]` instead of only `-`

Reply:
good idea
also added those to the _feature request acceptance criteria_, so the developer can check the items that were asked for and share their progress on the task more transparently (not sure that's what you had in mind though)